src/main/scala/com/codacy/parsers/implementation/CoberturaParser.scala

```diff
@@ -1,18 +1,41 @@
 package com.codacy.parsers.implementation
 
 import java.io.File
+import java.util.Locale
+import java.text.NumberFormat
 
 import com.codacy.api.{CoverageFileReport, CoverageReport, Language}
 import com.codacy.parsers.XMLCoverageParser
 import com.codacy.parsers.util.LanguageUtils
 
 import scala.xml.Node
+import scala.util.{Try, Success, Failure}
 
 class CoberturaParser(val language: Language.Value, val rootProject: File, val coverageReport: File) extends XMLCoverageParser {
 
   val rootProjectDir = rootProject.getAbsolutePath + File.separator
   lazy val allFiles = recursiveListFiles(rootProject)(f => f.getName.endsWith(LanguageUtils.getExtension(language)))
 
+  private[this] def convertToFloat(str: String): Float = {
+    Try {
+      str.toFloat
+    } match {
+      case Success(value) => value
+      case Failure(_) =>
+        Try{
+          // The french locale uses the comma as a sep.
+          val instance = NumberFormat.getInstance(Locale.FRANCE)
+          val number = instance.parse(str)
+          number.floatValue()
+        } match{
+          case Success(value) => value
+          case Failure(_) =>
+            // What we can do in this case?
```

Comment:
return a Try[Float]. if it can't parse string to float it should be the caller to deal with that, depending on what's best for it